### ISSUE TITLE
JOSS Review: where can I find the documentation?

### ISSUE THREAD

**Issue by Luthaf** [ToyotaResearchInstitute/materialnet#163]
I did not see a link in the Readme to the documentation of this package, or instructions about how to generate this documentation locally.

**Comment by waxlamp** [comment]
There is currently no standalone documentation; instead, the documentation exists as popovers that open when clicking the question-mark icons within the running app.

If you believe it's appropriate, I can add this explanation to the README file. Please let me know.

**Comment by Luthaf** [comment]
I think the popovers are fine for user documentation, together with #166.


Is there any kind of developer/API documentation? Potentially something automatically generated from comments in the code.

**Comment by waxlamp** [comment]
> I think the popovers are fine for user documentation, together with #166.
> 

Sounds good. We are working on #166 now.

> Is there any kind of developer/API documentation? Potentially something automatically generated from comments in the code.

Unfortunately, no. There is a bit of discussion in the README about some aspects of how the code is architected, but no formal documentation.

**Comment by Luthaf** [comment]
I think it would be worth the time and effort investment to create a developer documentation. I don't think this is blocking for the JOSS review, but adding developer/architectural documentation would improve the long term usefulness of MaterialNet 😃

**Comment by waxlamp** [comment]
> I think it would be worth the time and effort investment to create a developer documentation. I don't think this is blocking for the JOSS review, but adding developer/architectural documentation would improve the long term usefulness of MaterialNet smiley

I agree, and as we continue to work on it, we will attempt to generate some dev docs. It would definitely be a help to the project overall!
